examples/angular/src/main.html

```diff
@@ -0,0 +1,102 @@
+<div class="ds-content">
+  <ds-skip-nav href="#main"></ds-skip-nav>
+  <ds-usa-banner></ds-usa-banner>
+  <main id="main" class="ds-l-container">
+    <h1 class="ds-text-heading--4xl ds-u-margin-top--2">Web-component examples</h1>
+
+    <div class="ds-u-measure--base ds-u-padding-bottom--4">
+      <p>This is an example showing how to use our library of interactive web components.</p>
+
+      <ds-alert
+        heading="{{alertHeading}}"
+        [attr.variation]="alertVariation ?? null"
```

Comment:
Nit: does it make sense to coalesce a null value to null?

Reply:
It's the only way I can keep Angular from passing an empty string to that prop. That actually highlights an issue that I didn't want to try to fix in this PR but that should probably be discussed: It's actually not very easy for a downstream app to have the variation as a variable and pass it on, because the only way someone can make an info alert is if this is undefined. Our web component wrapper doesn't treat an empty string as undefined, either, which means the only way to pass on the lack of a variation is to not set the attribute in the first place. Two possible fixes:

1. Just change the `ds-alert` wrapper to interpret empty or nullish values as `undefined` when passing it on to the `Alert` React component or
2. Change the varations that the `Alert` React component accepts to include something like `"info"` as one of the options.

Reply:
Ah ok. Is a possible issue with option 2 that it would become a breaking change? In cases where users have no variation expressed they would have to supply an option?

Reply:
Interestingly there's an open PR (https://github.com/CMSgov/design-system/pull/3301) that is asking to expose 'informational' (or the Alert's default type)

Reply:
I was thinking about that. It's not a bad idea. Maybe `info` is a good abbreviation for it, and we might need to take a look at all the places where it's used and change some actual code and not just the types in order to merge that.

Reply:
@jack-ryan-nava-pbc, regarding it being a breaking change, I actually think we could make it a completely backwards-compatible change. Leaving it off would still default to `info` like it always has.